misc/python/materialize/cli/ci_coverage_pr_report.py

```diff
@@ -120,7 +123,7 @@ def mark_covered_lines(
                     coverage[file][line_nr] = (coverage[file][line_nr] or 0) + hit
 
 
-def get_report(coverage: Coverage) -> str:
+def get_report(coverage: Coverage) -> Tuple[str, str]:
```

Comment:
The two try blocks in this method look pretty similar (I think the diff is only the two additional parts of the if statement regarding the lines). Any chances of extracting a generic method to resolve this clone?

Reply:
Thanks, had more differences in the beginning, didn't realize I made them exactly the same :D